### PR TITLE
fix(DB/Creature): Add Fellicent path and spawntime

### PR DESCRIPTION
**Pull request by Ollie-Armitage**
## Changes Proposed:
-  Add path for Fellicent shade (counterclockwise around field)
-  Change spawntime for fellicent shade (5 minutes -> 90 minutes)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9079

## Tests Performed:
- Watched the full path, seems to be working. Windows 11.

## How to Test the Changes:

1. .go creature 42143
2. Click on 'Fellicent Shade' .
3. It should be moving in a counter clockwise fashion around the field.
4. It should respawn after 90 minutes.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1637134824779324732.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+UPDATE `creature`
+SET `spawntimesecs` = 5400
+WHERE `guid` = 42143;
+
+
+-- Insert new path
+
+UPDATE `creature_addon`
+SET `path_id` = @newPathId
+WHERE `guid` = 42143;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1637134824779324732.sql:16]:
```suggestion
UPDATE `creature_addon` SET `path_id` = @newPathId WHERE `guid` = 42143;
```

File: data/sql/updates/pending_db_world/rev_1637134824779324732.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1637134824779324732');
+
+SET @newPathId = 42143 * 10;
+
+-- Change Spawn time
+
+UPDATE `creature`
+SET `spawntimesecs` = 5400
+WHERE `guid` = 42143;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1637134824779324732.sql:9]:
```suggestion
UPDATE `creature` SET `spawntimesecs` = 5400 WHERE `guid` = 42143;
```

File: data/sql/updates/pending_db_world/rev_1637134824779324732.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+UPDATE `creature_addon`
+SET `path_id` = @newPathId
+WHERE `guid` = 42143;
+
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `delay`)
+VALUES
+       (@newPathId, 1, 1960, -346, 35.5772, 10000),
+       (@newPathId, 2, 1952, -421, 35.5772, 10000),
+       (@newPathId, 3, 2046, -419, 35.5772, 10000),
+       (@newPathId, 4, 2056, -357, 35.5772, 10000);
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1637134824779324732.sql:23]:
```suggestion
INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `delay`) VALUES
(@newPathId, 1, 1960, -346, 35.5772, 10000),
(@newPathId, 2, 1952, -421, 35.5772, 10000),
(@newPathId, 3, 2046, -419, 35.5772, 10000),
(@newPathId, 4, 2056, -357, 35.5772, 10000);
```

File: data/sql/updates/pending_db_world/rev_1637134824779324732.sql [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+SET @newPathId = 42143 * 10;
+
+-- Change Spawn time
+
+UPDATE `creature`
+SET `spawntimesecs` = 5400
+WHERE `guid` = 42143;
+
+
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1637134824779324732.sql:11]:
```suggestion
```

**Review by Nefertumm — CHANGES_REQUESTED** [review]

**Comment by Nefertumm** [comment]
Tested and it doesn't work. You forgot to update MovementType from either creature or creature_template. I will add it to my previous review in order to work.

File: data/sql/updates/pending_db_world/rev_1637134824779324732.sql (hunk 2 above)
Comment (Nefertumm) [reply data/sql/updates/pending_db_world/rev_1637134824779324732.sql:9]: Added movementType as it was missing

**Comment by Ollie-Armitage** [comment]
@Nefertumm Cheers, that's been committed.

**Review by Nefertumm — APPROVED** [review]

**Pull request merged**